inference_rules.adoc

```diff
@@ -370,12 +379,10 @@ The execution of LoadGen is restricted as follows:
 * The trace generated by LoadGen must be stored in the DRAM that most faithfully simulates queries arriving 
   from the most logical source, which is usually the network or an I/O device such as a camera. It may be pinned. 
   Similarly, the response provided to Loadgen must be stored in the DRAM that most faithfully simulates transfer 
-  to the most logical destination, which is a CPU process unless otherwise specified for the benchmark.
-
-  Submitters seeking to use anything other than the DRAM attached to the processor on which loadgen is running must 
-  seek prior approval, and must provide with their submission sufficient details system architecture and software to  
-  show how the I/O bandwidth utilized by each benchmark/scenario combination can be transferred between that memory and
-  the network or I/O device.
+  to the most logical destination, which is a CPU process unless otherwise specified for the benchmark. 
+  Submitters must provide with their submission sufficient details of the system architecture and software to  
+  show how the I/O bandwidth utilized by each benchmark/scenario combination can be transferred between the memory where the trace is stored and
+  the network or I/O device.  All components mentioned in the system architecture description must be populated during the run. 
```

Comment:
I dont think it makes sense to really enforce this requirement when loadgen is actually not using the networking interface. It would make sense if the server scenario is modified so that queries are indeed coming via the networking interface.

Reply:
Right, this was a WIP and we did discuss within the Power WG that this doesn't make sense.  Instead, we decided to make some of the System JSON fields more descriptive and better suited for power.  This change will affect only power submissions, and will not affect any other submissions like perf.-only or training perf.  

I'll close this PR shortly and reopen another one once the Power WG approves the changes.  Please wait for that update and review once I create the PR.